conversations.yaml

```diff
@@ -527,6 +577,16 @@ components:
                         type: string
                       message:
                         type: string
+    ErrorResponse:
+      properties:
+        statusCode:
+          type: integer
```

Comment:
Add a `description`.  Is statusCode something unique to Conversations?  Or is it the HTTP status code for reference?  Or something else?